.github/workflows/compose.yml

```diff
@@ -5,8 +5,12 @@ on: [push, pull_request]
 jobs:
   webui-docker-compose:
     runs-on: ubuntu-latest
-    timeout-minutes: 60
+    timeout-minutes: 650
     steps:
       - uses: actions/checkout@v4
-      - name: Verify that containers can be composed
-        run: make test-containers-compose
+      - uses: nick-fields/retry@v3
```

Comment:
Why not a retry within the code where it applies? Consider to explain that in your commit rather than (just) here

Reply:
#5772 is an alternative retrying from within the script

Reply:
#5772 looks better. merged meanwhile. Close here?